src/i18n/it.json

```diff
@@ -1,6 +1,6 @@
 {
   "__locale": "Italian (it)",
-  "__author": "davegraziosi",
+  "__author": "Giuseppe Lodi Rizzini",
```

Comment:
By courtesu please to not erase the name of previous contributors. You can put both names.